services/scheduler/src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -85,6 +88,7 @@
   private org.quartz.Scheduler quartzScheduler;
 
   private volatile boolean started = false;
+  private List<Scheduler> activeSchedulers = synchronizedList(new ArrayList<>());
```

Comment:
Why would a source not dispose its schedulers when the application it's in is disposed?